services/EnsService.ts

```diff
@@ -150,6 +163,23 @@ export default class EnsService {
     return this.textRecordValues;
   }
 
+  public IsBlueSkyRecordValid(record) {
+    // check if record has did:plc:
+    record = 'did:plc:ih4qsavxkbwgjeglbrdjocgg'
+
+    if(!record.includes('did:plc:')) {
```

Comment:
@makhnovskiy I would prefer it you make a regex validation instead of this... technically:
```
===sadawasd^&did:plc:sss
```

^ that would pass your validation, wouldn't it? And it would also pass the second validation I think, b/c `[1]` would only take the `sss` part.

That being said, I would prefer a single regex that validates the start of the string + the subsequent content (alpha-numeric limited to 24 chars) and validate the string as a whole thing,